docs/hosting-a-node.md

```diff
@@ -31,7 +31,7 @@ ___
 | CPU       | 2 Core                | 4 Core                    |
 | RAM       | 8 GB                  | 16 GB                     |
 | Bandwidth | 10 Mbit               | 20 Mbit                   |
-| Disk      | 500 GB SSD            | 1 TB fast NVMe SSD        |
+| Disk      | 500 GB NVMe SSD            | 1 TB NVMe SSD        |
```

Comment:
Set masternode's disk type to all NVMe SSDs but kept public node untouched.

Reply:
Actually, with current recommended settings, a full node only needs ~100GB, given extra space for growing up, 300GB is enough for the minimum and we can adjust the recommend to 500GB, wdyt? @darrenvechain

Reply:
We have this one from vechain stats, it says a full archive node uses ~400

https://vechainstats.com/charts/#thor-size

Reply:
Masternode is recommended to run in full node, not full archive node. Should we mention that as well?

Reply:
I guess anything which help users we should mention

Reply:
Ya I think these recommendations came from @kgapos based on running full archive nodes